public/app/features/dashboard/components/DashboardSettings/AutoRefreshIntervals.test.tsx

```diff
@@ -4,7 +4,7 @@ import userEvent from '@testing-library/user-event';
 import { defaultIntervals } from '@grafana/ui/src/components/RefreshPicker/RefreshPicker';
 
 import { AutoRefreshIntervals, getValidIntervals, Props, validateIntervals } from './AutoRefreshIntervals';
-import { TimeSrv } from 'app/core/services/TimeSrv';
+import { TimeSrv } from '../../services/TimeSrv';
```

Comment:
My bad @hugohaggmark, it's `app/features/dashboard/services/TimeSrv`, sorry about that!

Reply:
No worries :)